app/views/admin/responses/_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= form_for [:admin, consultation, consultation_response], url: [:admin, consultation, consultation_response.singular_routing_symbol] do |form| %>
+  <%= render "govuk_publishing_components/components/fieldset", {
+    legend_text: "Published on",
+    heading_size: "s"
+  } do %>
+    <%= render "components/datetime_fields", {
+      prefix: "consultation_#{consultation_response.singular_routing_symbol}",
+      field_name: "published_on",
+      # date_heading: "Published on",
+      date_only: true,
```

Comment:
we need to link errors here so we need to add 

```
error_items: errors_for(@consultation_response.errors, :published_at
```

It should also have a large heading

```
    heading_size: "l",
```